src/test/java/org/kiwiproject/test/dropwizard/app/PostgresAppTestExtensionTest.java

```diff
@@ -25,7 +25,6 @@
  * and https://github.com/zonkyio/embedded-postgres/issues/40 for more details.
  */
 @DisplayName("PostgresAppTestExtension")
-@EnabledOnOs(OS.LINUX)
```

Comment:
Please put this back. It still fails on my Mac, whether I run via IntelliJ or Maven on command line.

Reply:
...or are you saying that it fails b/c the `afterAll` was not there???

Reply:
Still fails when running via IntelliJ, and it _appears_ that it works when running via Maven, but I see this output:

```
[INFO] Running org.kiwiproject.test.dropwizard.app.PostgresAppTestExtensionTest
INFO  [2020-09-02 14:33:25,954] io.zonky.test.db.postgres.embedded.EmbeddedPostgres: Detected a Darwin x86_64 system
INFO  [2020-09-02 14:33:25,956] io.zonky.test.db.postgres.embedded.DefaultPostgresBinaryResolver: Detected distribution: 'Unknown'
INFO  [2020-09-02 14:33:25,959] io.zonky.test.db.postgres.embedded.DefaultPostgresBinaryResolver: System specific postgres binaries found: postgres-darwin-x86_64.txz
INFO  [2020-09-02 14:33:26,107] io.zonky.test.db.postgres.embedded.EmbeddedPostgres: Postgres binaries at /var/folders/l0/0m9zr4j536n7gqw0z5dwrdsm0000gp/T/embedded-pg/PG-2c0c4f55a015ab253337e3d05ca5fd21
INFO  [2020-09-02 14:33:26,131] io.zonky.test.db.postgres.embedded.EmbeddedPostgres: dyld: Library not loaded: /usr/local/opt/openssl/lib/libssl.1.0.0.dylib
INFO  [2020-09-02 14:33:26,132] io.zonky.test.db.postgres.embedded.EmbeddedPostgres:   Referenced from: /usr/local/Cellar/postgresql/10.6/lib/libpq.5.10.dylib
INFO  [2020-09-02 14:33:26,132] io.zonky.test.db.postgres.embedded.EmbeddedPostgres:   Reason: image not found
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.262 s - in org.kiwiproject.test.dropwizard.app.PostgresAppTestExtensionTest
```

As you can see,  _no tests actually ran_ and we get that "Reason: image not found" message

Reply:
Ha, that was a problem with my local homebrew installation and not having openssl 1.0.0.

But, we decided for now to leave this enabled only on Linux. Added new issue #80 to deal with it later...